insights/parsers/numastat.py

```diff
@@ -0,0 +1,335 @@
+"""
+Command ``numactl`` and ``numastat`` parser
+=============================================
+
+Shared parsers for parsing commands ``numastat``, ``numastat -m`` and
+``numactl --hardware``.
+
+Parsers included in this module are:
+
+NUMAStat - command ``numastat``
+-------------------------------
+
+NUMAStatMem - command ``numastat -m``
+--------------------------------------
+
+NUMActlHw - command ``numactl --hardware``
+------------------------------------------
+"""
+
+from .. import parser, CommandParser
+from ..parsers.meminfo import MemInfo, AnonMemInfo, FileMemInfo, SlabMemInfo, HugePageMemInfo
+from insights.specs import Specs
+import re
+from os import linesep
+
+
+@parser(Specs.numastat)
+class NUMAStat(CommandParser):
```

Comment:
Instead of using a generic `data` attribute here consider deriving the parser from dict `NUMAStat(CommandParser, dict)` , and store the information directly in the parser object.  Here's an [example parser](https://github.com/RedHatInsights/insights-core/blob/625c82500cdcacc20bbd2d23e96d44e7fefa27cb/insights/parsers/ipsec_conf.py#L19) based on dict.  You may still add other attributes like `nodes` and properties.